### ISSUE TITLE
No landscape layout

### ISSUE THREAD

**Issue by adriantache** [adriantache/Bowling-Scoring#2]

**Comment by bytao7mao** [comment]
Hi @adriantache, i can gladly help you out with this issue .

**Comment by adriantache** [comment]
Awesome! Please submit a pull request when you're done :)

**Comment by bytao7mao** [comment]
@adriantache i have a question, do you want to keep the logo from the top of the app in landscape mode ? because it doesn't have enough 'room' for the logo.
Also another issue it is that, when i turn the device to landscape mode, the app is not going into landscape mode... but i will make the pull request after more tests.

**Comment by adriantache** [comment]
@bytao7mao Not necessarily, no. And it doesn't switch to landscape because AndroidManifest.xml is preventing the activity to change orientation.
